extension.js

```diff
@@ -45,14 +43,23 @@ const activate = async (context) => {
 
 		setInterval(() => {
 			const lapTime = new Date();
-
 			const incrementOfTime = Math.round((lapTime - startTime) / 60000);
-			
 
 			vscode.window.showInformationMessage(`You have been working for ${incrementOfTime} minutes. (This pops up every three seconds...Enjoy!)`);
 		}, 3000)
 	})
 
+	    // THIS IS WHERE STOP WATCH USER COMMAND FUNCTIONS BEGIN
+
+    let resetTime = vscode.commands.registerCommand('be-human.resetTime', function () {
+		 startTime = new Date();
+		 return resetTime
```

Comment:
It looks like the line above is getting a new startTime, but this function is returning a resetTime. I believe the line above (55) was intended to be:
const resetTime = new Date();

Or line 56 could be:
return startTime

Also, does this stop the pop-up that is already running? Or would there now be two pop-ups running? The original startTime pop-up and this new resetTime pop-up.

Might it be possible to simply call the startTime function again? Would that reset it? Or would that simply create a second startTime pop-up running on a different time tracker than the first startTime pop-up?

Reply:
Thank you Franco! The variable is called reset time because it creates a user terminal command with a different title than startTime. The function does the same thing, but was made so that the user can type "resetTime" in the terminal to reset the timer.